README.md

```diff
@@ -71,6 +71,7 @@ Below is the expected structure of such a config file:
   "vow_dump": "<Flop initial lot size in MKR unit>",
   "vow_bump": "<Flap fixed lot size in DAI unit>",
   "vow_hump": "<Flap Surplus buffer in DAI unit>",
+  "cat_box": "<Max total amount on auctions in DAI unit>",
```

Comment:
I would be more precise:

`<Max total DAI needed to cover all debt plus penalty fees on active auctions in DAI units>`

Reply:
Solved.